modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/action/EditEntryMVCActionCommand.java

```diff
@@ -537,7 +537,10 @@ private BlogsEntry _updateEntry(ActionRequest actionRequest)
 		ServiceContext serviceContext = ServiceContextFactory.getInstance(
 			BlogsEntry.class.getName(), actionRequest);
 
-		if (_featureFlagManager.isEnabled("LPD-11147")) {
+		if ((serviceContext != null) &&
+			_featureFlagManager.isEnabled(
+				serviceContext.getCompanyId(), "LPD-11147")) {
+
```

Comment:
Why are you checking the `serviceContext` for nullness? Also, you already got the `themeDisplay` in scope, so you could get the company ID from there.

Reply:
thanks, I will change it.